openwisp_ipam/tests/test_api.py

```diff
@@ -251,6 +251,24 @@ def test_hosts_list_api(self):
         self.assertEqual(self.client.get(response.data['previous']).status_code, 200)
         self.assertIsNone(response.data['next'])
 
+    def test_bearer_auth(self):
```

Comment:
let ensure we only using the bearer for authentication by first loging out. this can be achieved with `self._logout()`